CONTRIBUTING.md

```diff
@@ -136,6 +136,25 @@ npx shadcn@latest add "https://magicui.design/r/example-component"
 
 <ComponentSource name="example-component" />
 
+<Step>Update the import paths to match your project setup.</Step>
+
+<Step>Add the required CSS animations</Step>
+
+<Step>Add the following animations to your global CSS file inside the `@theme inline` block (e.g., `app/globals.css` or similar)</Step>
```

Comment:
this should have the `@theme inline` block in the ```code example``` below instad of mentioning above the code example


also we can use line highlighting

```tsx {1-4}{5-10}
line highlght
```